### PR TITLE
fix: cheerio import and yt scrape data

### PR DESCRIPTION
**Pull request by devansh-m12**
This fixes https://github.com/solana-foundation/solana-com/issues/58

here the proof : 
Screenshot 2024-10-07 at 1 18 00 PM

### CODE REVIEW

**Comment by devansh-m12** [comment]
We're already using Google's APIs to fetch YouTube playlists and videos, so why not use the same APIs to get the subscriber count instead of scraping the data, which avoids the uncertainties of updates?

File: src/utils/followerFunctions.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import cheerio from "cheerio";
+import * as cheerio from "cheerio";
```
Comment (catalinred) [inline src/utils/followerFunctions.js:1]: Are these change needed if `scrapeYoutubeSubscriberCount` is not used anymore?

If `scrapeYoutubeSubscriberCount` is getting superseded by `getYoutubeSubscriberCount` let's remove the former function completely

File: src/utils/followerFunctions.js [hunk 2]
```diff
@@ -54,17 +54,45 @@ const scrapeYoutubeSubscriberCount = async () => {
...
   }
   return 0;
 };
 
+const getYoutubeSubscriberCount = async () => {
+  const channelId = process.env.YOUTUBE_CHANNEL_ID;
+  const YOUTUBE_API_KEY = process.env.YOUTUBE_API_KEY;
+  try {
+    const response = await fetch(
+      `https://www.googleapis.com/youtube/v3/channels?part=statistics&id=${channelId}&key=${YOUTUBE_API_KEY}`,
```
Comment (catalinred) [inline src/utils/followerFunctions.js]: Can you move `baseURL` outside as global const to use in both `getYTVideos` and newly added `getYoutubeSubscriberCount `?

**Review by catalinred — CHANGES_REQUESTED** [review]
Thanks for working on this @devansh-m12
This looks good overall, I added a couple of comments below

File: src/utils/followerFunctions.js (hunk 1 above)
Comment (devansh-m12) [reply src/utils/followerFunctions.js:1]: These changes are essential for `scrapeUrlForTag`, which is used to extract both the `Meetup member count` and `VK follower count`. As seen in the image, the fixes for the `Meetup member count` are now functioning correctly. However, there is still an issue with the `VK follower count`, which I plan to address in a separate PR. Additionally, as per your instructions, I will remove the `scrapeYoutubeSubscriberCount` function.

**Review by catalinred — APPROVED** [review]
Lgtm, thanks for working on it!

**Comment by devansh-m12** [comment]
@catalinred Thanks! On a separate note, the VK followers count isn't working, and it seems like much of the data in the community stats section is hardcoded. I'd be interested in working on that.

**Comment by catalinred** [comment]
@devansh-m12 I will have this branch merged next and fwiw I just added the #hacktoberfest topic label to this repo ;) I think this initiative is great, thanks for the suggestion.

Some of the community stats are hardcoded and could definitely use some attention. Feel free to open an issue or submit a PR, and we can continue the discussion there on how to address it.
